main.tf

```diff
@@ -1,6 +1,7 @@
 resource "castai_gke_cluster" "castai_cluster" {
   project_id                 = var.project_id
   region                     = var.gke_cluster_region
+  location                   = var.gke_cluster_location != null ? var.gke_cluster_location : var.gke_cluster_region
```

Comment:
Is region not redundant then if we have location already?

Reply:
Region removed. 

`gke_cluster_region` I'm not removing this variable for backward compatibility. WDYT? Should I remove it?

Reply:
Ok, maybe mark if as deprecated and we can leave as is.